server/src/lib.rs

```diff
@@ -576,6 +662,28 @@ mod tests {
         let read_rules = serde_json::from_str::<DatabaseRules>(read_data).unwrap();
 
         assert_eq!(rules, read_rules);
+
+        let db2 = "db_awesome";
+        server
+            .create_database(db2, DatabaseRules::default())
+            .await
+            .expect("failed to create 2nd db");
+
+        println!(
+            "prefixes = {:?}",
+            store
+                .list_with_delimiter(&ObjectStorePath::from_cloud_unchecked(""))
+                .await
+                .unwrap()
+        );
```

Comment:
I'm not a fan of having `println!` in tests. 

The happy path is for Rust to completely hide `stdout` from the user (or from a program processing test output). So printing to `stdout` only has value when a test fails. It is preferable (in my opinion) to use `assert` and friends to provide this extra context when a test fails.

An alternative is: 

```suggestion
        let db2 = "db_awesome";
        assert!(
            server
                .create_database(db2, DatabaseRules::default())
                .await
                .is_ok(),
            "failed to create 2nd db. prefixes = {:?}",
            store
                .list_with_delimiter(&ObjectStorePath::from_cloud_unchecked(""))
                .await
                .unwrap()
        );
```

Reply:
Ah, that was cruft I accidentally left behind. Removed in b3454daf7097d7f70a535b5ff09b6f75caea3dd0